lowstepper_wasm_demo/src/styles.css

```diff
@@ -1,28 +1,261 @@
-html, body {
-  margin: 0;
-  padding: 0;
-}
-
-.App {
-
-  font-family: sans-serif;
-  text-align: center;
-}
-
-p {
-  font-size: 16px;
-}
-
-.container {
-  width: 440px;
-  display: flex;
-  justify-content: center;
-  flex-direction: column;
-  align-items: center;
-  margin: 0 auto;
-  margin-top:20px;
-}
-.container > * {
-  padding: 20px;
-
-}
+html, body {
+  margin: 0;
+  padding: 0;
+}
+
+.App {
+
+  font-family: sans-serif;
+  text-align: center;
+}
+
+p {
+  font-size: 16px;
+}
+
+.container {
+  width: 440px;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  
+  margin: 0 auto;
+  margin-top:20px;
+}
+.container > * {
+  padding: 20px;
+}
+
+#root {
+	background-color:#000000;
+	margin:-20px auto 0px;
+	max-width:596px;
+	position:relative;
+}
+
+	#root .guide{
+		display: none;
+		left:0px;
+		opacity:0.5;
+		position:absolute;
+		top:0px;
+	}
+
+	#root h1{
+		color:#fff;
+		padding:10px;
+		margin-bottom: 10px;
+	}
+
+	#root .white_hole{	
+		position:absolute;

```

Comment:
you legend lol

I would suggest doing it with flexbox... but I can't imagine that's very easy. This seems to work fine.